search/search-client-elasticsearch/src/main/java/io/camunda/search/es/clients/ElasticsearchSearchClient.java

```diff
@@ -238,4 +239,12 @@ private static CamundaSearchException.Reason searchExceptionToReason(final Excep
     }
     return CamundaSearchException.Reason.SEARCH_CLIENT_FAILED;
   }
+
+  static void logException(String msg, Exception exception) {
+    if (exception instanceof SocketTimeoutException) {
```

Comment:
❓ Should this also go into `searchExceptionToReason`? This is a CONNECTION_FAILED reason, isn't it?